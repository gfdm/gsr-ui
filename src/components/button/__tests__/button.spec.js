import { mount, createLocalVue } from '@vue/test-utils'
import IdButton from '../'

const localVue = createLocalVue()
localVue.use(IdButton)

describe('Button', () => {
  test('should render well', () => {
    const Wrapper = {
      name: 'wrap',
      template: `
        <id-button>Click Me!</id-button>
      `
    }

    const wrapper = mount(Wrapper, {
      localVue
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
