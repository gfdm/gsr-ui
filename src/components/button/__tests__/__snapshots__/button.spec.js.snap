// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render well 1`] = `<button class="id-button id-button--primary"><span>Click Me!</span></button>`;
